lib/linux_ccnt/aarch64_cycle_counter.c

```diff
@@ -0,0 +1,86 @@
+/*- C -*- */
+/*
+ *
+ *    Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ * 
+ *    For any questions about this software or licensing,
+ *    please email opensource@seagate.com or cortx-questions@seagate.com.
+ */
+
+
+#define M0_TRACE_SUBSYSTEM M0_TRACE_SUBSYS_LIB
+
+#include "lib/trace.h"
+/*** Header includes are after this line. Remove this line. ***/
```

Comment:
We need to remove this line, just as it says.

Reply:
can u explain it  "/*** Header includes are after this line. Remove this line. ***/". what does it mean

Reply:
So, you see, this is a template file.
In every *.c file, the first header file included is 
```
#define M0_TRACE_SUBSYSTEM M0_TRACE_SUBSYS_LIB
#include "lib/trace.h"
```
and then other head header files are followed.
This comment is a marker.
Other files should be after this line.
And then this line should be removed.